vector/src/main/java/fr/gouv/tchap/util/DinsicUtils.java

```diff
@@ -334,6 +335,81 @@ public void onUnexpectedError(final Exception e) {
         return succeeded;
     }
 
+    /**
+     * Handle the click on a contact in the case of a direct chat
```

Comment:
I would update the description here with:
Prepare a dialogue with a selected contact.

About the third param ("item"), we should rename it: "selectedContact", We should not mention "list of the contacts" here. I would describe this param like "the selected contact"